src/util/util.js

```diff
@@ -47,8 +49,9 @@ function resolveURLTemplates(URLTemplates, variables = {}, options = {}) {
   // RANDOM/random is not defined in VAST 3/4 as a valid macro tho it's used by some adServer (Auditude)
   variables['RANDOM'] = variables['random'] = variables['CACHEBUSTING'];
 
-  for (const URLTemplateKey in URLTemplates) {
-    let resolveURL = URLTemplates[URLTemplateKey];
+  const URLArray = extractURLsFromTemplates(URLTemplates);
```

Comment:
just a minor comment, I would move this to the top of the function, so it is clear we shouldn't use `URLTemplates` directly

Reply:
Done